modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- In some cases `scrollToMarker` make the entire page scroll. #TINY-9828
```

Comment:
Is the `scrollToMarker` the public API to mention in the changelog?

Reply:
no, the user action is pressing enter. That's all the changelog needs to mention.

Reply:
changed